<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sale_management_system</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_product">
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>200</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="man_pro">
      <attribute name="title">
       <string>제품 관리 (&amp;1)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="lb_pro_name">
             <property name="text">
              <string>제품명    </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_pro_name"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="lb_category">
             <property name="text">
              <string>종류</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_menu"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="lb_pro_code">
           <property name="text">
            <string>제품 코드</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_pro_code"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="lb_pro_price">
           <property name="text">
            <string>제품 단가</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_pro_price"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="lb_pro_margin">
           <property name="text">
            <string>마진율    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_pro_margin"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QPushButton" name="btn_insert_menu">
           <property name="text">
            <string>메뉴 등록 (&amp;i)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_edit_menu">
           <property name="text">
            <string>메뉴 편집 (&amp;e)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ins_category">
           <property name="text">
            <string>종류 추가 (&amp;a)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_del_menu">
           <property name="text">
            <string>메뉴 제거 (&amp;d)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="man_sale">
      <attribute name="title">
       <string>판매 관리 (&amp;2)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lbl_category">
               <property name="text">
                <string>제품 유형</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_category"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lbl_pro_name">
               <property name="text">
                <string>제품명</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_pro_name"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lbl_salecount">
             <property name="text">
              <string>판매량</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_salecnt"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btn_insert">
           <property name="text">
            <string>메뉴추가 (&amp;a)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_edit">
           <property name="text">
            <string>편집 (&amp;e)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_delete">
           <property name="text">
            <string>삭제 (&amp;d)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_exit">
           <property name="text">
            <string>종료 (&amp;x)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QPushButton" name="btn_ok">
           <property name="text">
            <string>주문완료 (&amp;o)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reset">
           <property name="text">
            <string>초기화 (&amp;r)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sel_history">
      <attribute name="title">
       <string>검색 (&amp;3)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="btn_select">
           <property name="text">
            <string>검색 (&amp;s)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_init">
           <property name="text">
            <string>검색 초기화 (&amp;i)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="lbl_pro_name_2">
             <property name="text">
              <string>제품명   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_sel_pro"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>가격       </string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLineEdit" name="le_min_price">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>원</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>~</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_max_price">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>원</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="lbl_time">
               <property name="text">
                <string>시간       </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_min_date_y"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>년</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_min_date_m"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>월</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_min_date_d"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>일</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>~</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_max_date_y"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>년</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_max_date_m"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>월</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_max_date_d"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>일</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lb_order">
       <property name="text">
        <string>주문현황</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="table_sale">
         <property name="columnCount">
          <number>4</number>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="table_order">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
